merge/update.go

```diff
@@ -57,10 +58,18 @@ func (s *Updater) update(oldObject, newObject typed.TypedValue, version fieldpat
 			var err error
 			versioned.oldObject, err = s.Converter.Convert(oldObject, managerSet.APIVersion)
 			if err != nil {
+				if s.Converter.IsMissingVersionError(err) {
+					delete(managers, manager)
+					continue
+				}
 				return nil, fmt.Errorf("failed to convert old object: %v", err)
 			}
 			versioned.newObject, err = s.Converter.Convert(newObject, managerSet.APIVersion)
 			if err != nil {
+				if s.Converter.IsMissingVersionError(err) {
```

Comment:
I don't know how useful this is. Most of the time, if we managed to convert the oldObject, we shouldn't fail to convert the new object right after because the same destination error doesn't exist. BUT, with CRDs, I wonder if there could be a race condition.